installer/IP35_uefi_extraction.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+# EMBA - EMBEDDED LINUX ANALYZER
+#
+# Copyright 2020-2022 Siemens Energy AG
+#
+# EMBA comes with ABSOLUTELY NO WARRANTY. This is free software, and you are
+# welcome to redistribute it under the terms of the GNU General Public License.
+# See LICENSE file for usage of this software.
+#
+# EMBA is licensed under GPLv3
+#
+# Author(s): Benedikt Kuehne
+
+# Description:  Installs UEFIExtract from https://github.com/LongSoft/UEFITool/releases
+#               into external/UEFITool/UEFIExtract
+
+IP35_uefi_extraction() {
+  module_title "${FUNCNAME[0]}"
+  
+  
+  INSTALL_APP_LIST=()
+
+  print_file_info "UEFIExtract_NE_A62_linux_x86_64.zip" "Release-version A62" "https://github.com/LongSoft/UEFITool/releases/download/A62/UEFIExtract_NE_A62_linux_x86_64.zip" "external/UEFITool/UEFIExtract_NE_A62_linux_x86_64.zip"
+  print_tool_info "unzip" 1
+
+  if [[ "$LIST_DEP" -eq 1 ]] || [[ $DOCKER_SETUP -eq 0 ]]; then
```

Comment:
DOCKER_SETUP is for building the container. Your check bypasses this and the new dependency will not get installed in the container.
I guess this should be something like this:
```
    if [[ "$LIST_DEP" -eq 1 ]] || [[ $DOCKER_SETUP -eq 1 ]] ; then
```